types/hedron/index.d.ts

```diff
@@ -1,172 +1,147 @@
-// Type definitions for Hedron 0.7
-// Project: https://github.com/JSBros/hedron
-// Definitions by: Dmytro Borysov <https://github.com/dborysov>
+// Type definitions for Hedron 1.0
+// Project: https://github.com/garetmckinley/hedron
+// Definitions by: James Best <https://github.com/jim-at-jibba>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
+// TypeScript Version: 3.3
 
-import * as React from 'react';
+import * as React from "react";
 
-export interface ColumnProps {
-    /**
-     * You can specify component to use specific tag
-     * @default 'div'
-     */
-    tagName?: keyof ElementTagNameMap;
-    /**
-     * Draws all child columns with "bounding boxes" for easy
-     * visualization of the grid. This enables debug mode for all the
-     * children of this component
-     * @default false
-     */
-    debug?: boolean;
-    /**
-     * The amount of horizontal columns this row creates.
-     * @default 12
-     */
-    divisions?: number;
-    /**
-     * If true, disable padding.
-     */
-    fluid?: boolean;
-    /**
-     * Width during xs breakpoint.
-     */
-    xs?: number;
-    /**
-     * Width during sm breakpoint.
-     */
-    sm?: number;
-    /**
-     * Width during md breakpoint.
-     */
-    md?: number;
-    /**
-     * Width during lg breakpoint.
-     */
-    lg?: number;
-    /**
-     * Width of left margin during xs breakpoint.
-     */
-    xsShift?: number;
-    /**
-     * Width of left margin during sm breakpoint.
-     */
-    smShift?: number;
-    /**
-     * Width of left margin during md breakpoint.
-     */
-    mdShift?: number;
-    /**
-     * Width of left margin during lg breakpoint.
-     */
-    lgShift?: number;
+export interface Styleable {
+  /**
+   * The amount of padding added
+   */
+  padding?: string;
+  /**
+   * The amount of padding added
+   */
+  margin?: string;
+  /**
+   * Width Property
+   */
+  width?: string;
+  /**
+   * Height Property
+   */
+  height?: string;
+  /**
+   * Visibilty Property
+   */
+  visibility?: "visible" | "hidden" | "collapse" | "initial" | "inherit";
+  /**
+   * Display Property
+   */
+  display?: string;
```

Comment:
I think it should be a type union instead of `string`?

```suggestion
  display?:
    | 'inline'
    | 'block'
    | 'contents'
    | 'flex'
    | 'grid'
    | 'inline-block'
    | 'none'
    | 'initial'
    | 'inherit'
    | 'inline-flex'
    | 'inline-grid'
    | 'inline-table'
    | 'list-item'
    | 'run-in'
    | 'table'
    | 'table-caption'
    | 'table-column-group'
    | 'table-header-group'
    | 'table-footer-group'
    | 'table-row-group'
    | 'table-cell'
    | 'table-column'
    | 'table-row';
```

Reply:
Yep definitely. Will update

Reply:
Noted, will update